### PR TITLE
Prevent to start filebeat when STDIN is used with other inputs

### PR DESCRIPTION
**Pull request by ph**
- **breaking changes:** Stdin will now requires exclusive mode to run.
- Added helpers on top of the config class to retrieve enabled plugins.
- Refactored the stdin python integration testing into their own file.
- Added tests to check the combination of STDIN with other plugins
- Loop once on the enabled inputs

### CODE REVIEW

**Comment by ph** [comment]
Due to the nature of stdin related to EOF, it more appropriate to make sure the Stdin input is run in exclusive mode.

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -47,6 +47,8 @@ https://github.com/elastic/beats/compare/v6.0.0-beta2...master[Check the HEAD di
 - Refactor the usage of prospector to input in the YAML reference and
   the system test. {pull}6121[6121]
 - Add IIS module to parse access log and error log. {pull}6127[6127]
+- Add validation for Stdin, when Filebeat is configured with Stdin and any other inputs, Filebeat
```
Comment (ruflin) [inline CHANGELOG.asciidoc]: I wonder if this is a bug fix or a breaking change :-)

**Comment by ruflin** [comment]
@ph We should add a note about this also to the docs.

**Comment by ph** [comment]
@ruflin I need to take a look at config reload, maybe we should do some logic to prevent it there too can be a separate PR.

File: CHANGELOG.asciidoc (hunk 1 above)
Comment (ph) [reply CHANGELOG.asciidoc]: Its kind of both 🍁

**Comment by ph** [comment]
I have added a note to the documentation.

@tsg could you take a look?

File: filebeat/config/config_test.go [hunk 2]
```diff
@@ -95,3 +96,81 @@ func TestMergeConfigFiles(t *testing.T) {
...
+		tests := []struct {
+			name     string
+			input    string
+			expected bool
+			config   *Config
+		}{
+			{name: "input exists and enabled", input: "stdin", expected: true, config: config},
+			{name: "input exists and disabled", input: "udp", expected: false, config: config},
+			{name: "input doesn't exist", input: "redis", expected: false, config: config},
+			{name: "not inputs are enabled", input: "redis", expected: false, config: &Config{}},
```
Comment (ruflin) [inline filebeat/config/config_test.go]: s/not/no/

File: filebeat/docs/inputs/input-stdin.asciidoc [hunk 3]
```diff
@@ -9,6 +9,8 @@
 
 Use the `stdin` input to read events from standard in.
 
+Note: This input cannot be run at the same time of other inputs type.
```
Comment (ruflin) [inline filebeat/docs/inputs/input-stdin.asciidoc]: at the same time "with" other input types?  (plural types, singluar input?)

**Comment by ph** [comment]
@ruflin typos fixed, rebased with master.

**Pull request merged**